DigitalLearningSolutions.Web/Controllers/TrackingSystem/Delegates/ViewDelegateController.cs

```diff
@@ -84,5 +84,21 @@ public IActionResult DeactivateDelegate(int delegateId)
 
             return RedirectToAction("Index", new { delegateId } );
         }
+
+        [HttpPost]
+        [Route("ReactivateDelegate")]
+        public IActionResult ReactivateDelegate(int delegateId)
+        {
+            var centreId = User.GetCentreId();
+            var delegateUser = userDataService.GetDelegateUserCardById(delegateId);
+            if (delegateUser == null || delegateUser.CentreId != centreId)

```

Comment:
I think this condition can be simplified to
`delegateUser?.CentreId != centreId`

Reply:
yes, thanks for suggestion. I thought it would need to be consistent with other methods. I have updated the code to simplify this check